.github/workflows/fmutools-publish-pypi.yml

```diff
@@ -1,31 +1,36 @@
 name: Publish Python 🐍 distributions 📦 to PyPI
 
 on:
+  pull_request:
+    branches: [main]
```

Comment:
With the conditional `if: github.event_name == 'release'` on the job configuration will this do anything? I think since this is a pure Python package it would be OK to skip a build on PR

Reply:
I will edit